doc/wave_calib.rst

```diff
@@ -44,14 +48,131 @@ be certain by adding the following to the :doc:`pypeit_file`
           clip = False
           subtract_continuum = True
 
-For a multislit observation, you should keep clip=False, and
-change subtract_continuum=True to subtract_continuum=False.
+For a multislit observation, you should keep ``clip = False``, and
+change ``subtract_continuum = True`` to ``subtract_continuum = False``.
+
+
+Line Lists
+==========
+
+PypeIt-Included Line Lists
+--------------------------
+
+Without exception, arc line wavelengths are taken from
+the `NIST database <https://physics.nist.gov/PhysRefData/ASD/lines_form.html>`_,
+*in vacuum*. These data are stored as ASCII tables in the
+`"arc_lines" directory <https://github.com/pypeit/PypeIt/tree/release/pypeit/data/arc_lines/lists>`_
+of the repository. Here are the available lamps:
+
+======  ==========  ================
+Lamp    Range (Å)   Last updated
+======  ==========  ================
+ArI     3100-11000  7 October 2018
+CdI     3000-6500   28 February 2022
+CuI     4200-6100   4 October 2018
+FeI     3000-10000  26 April 2020
+HeI     3800-6000   21 December 2016
+HgI     2900-12000  28 February 2022
+KrI     4000-10000  3 May 2018
+NeI     5000-12000  3 May 2018
+XeI     4000-12000  3 May 2018
+ZnI     3000-5000   21 December 2016
+ThAr    3000-11000  9 January 2018
+======  ==========  ================
+
+In the case of the ThAr list, all of the lines are taken from
+the NIST database, and are labelled with a 'MURPHY' flag if the
+line also appears in the list of lines identified by
+`Murphy et al. (2007) MNRAS 378 221 <http://adsabs.harvard.edu/abs/2007MNRAS.378..221M>`_
+
+
+User-Supplied Line Lists
+------------------------
+
+Occasionally users reliably find lines in their arc spectra that are
+not included in the lists above.  For experimenting with adding
+particular lines or for instrument-specific arc line detections,
+PypeIt now allows the use of user-supplied arc line lists for
+wavelength calibration.  The ability to use arbitrary line lists is
+included *caveat emptor*, and users MUST ensure that ALL lists used
+have wavelength measurements **in vacuum**.
+
+.. note::
+
+  Users who are confident that their new lines would benefit PypeIt
+  broadly (*i.e.*, beyond this single instrument) should consult with
+  the Developers.
```

Comment:
Maybe we should just encourage them to submit a PR?  The "consulting with the Developers" can happen in the PR comments.

Reply:
Adding encouragement.